python/grass/script/__init__.py

```diff
@@ -1,12 +1,15 @@
 """Python interface to launch GRASS GIS modules in scripts
 """
 
+from . import setup  # noqa: F401
```

Comment:
Why this?

Reply:
This was already there so I didn't change it. I'm assuming for some dependencies since it's not used directly

Reply:
It was me like a couple weeks ago. Ruff, in preview mode, didn't do the right suggestion, or had a suggestion loop of some sort. Or maybe the fix was worse with another tool, like pyright, as the file is named setup.py, and it's not a setuptools setup.py file.


So, you might want to try again, it wasn't a big bug.